Modules/FIT/FDD/src/PostProcTask.cxx

```diff
@@ -306,13 +306,13 @@ void PostProcTask::update(Trigger t, framework::ServiceRegistryRef)
   }
 
   auto mo3 = mDatabase->retrieveMO(mPathDigitQcTask, "AmpPerChannel", t.timestamp, t.activity);
-  auto hAmpPerChannel = mo3 ? dynamic_cast<TH2D*>(mo3->getObject()) : nullptr;
+  auto hAmpPerChannel = mo3 ? dynamic_cast<TH2F*>(mo3->getObject()) : nullptr;
```

Comment:
Why not `TH2`, so it works for both?

Reply:
I think better to cast into exact type of this object.

Reply:
I hope we do not get to a case where QC objects are produced with a QC version with `TH2D`, but the checks are applied with a newer QC version...